src/Client.php

```diff
@@ -1184,7 +1184,7 @@ protected function prepareHeaders($body, $uri)
         // Set the Accept-encoding header if not set - depending on whether
         // zlib is available or not.
         if (! $this->getRequest()->getHeaders()->has('Accept-Encoding')) {
-            if (function_exists('gzinflate')) {
+            if (empty($this->config['outputstream']) && function_exists('gzinflate')) {
```

Comment:
I'm curious: can you explain why this is needed?

Reply:
See the test - we'd like to have uncompressed content stored in the file if we are using streams and we set the file.